### ISSUE TITLE
Misleading error for Lombok plugin on Android Studio

### ISSUE THREAD

**Issue by dsjstc** [mplushnikov/lombok-intellij-plugin#264]
Just updated the Lombok plugin to 0.12.15 in Android Studio 2.1.  I get the following misleading error at startup.  AFAIK, Android Studio 2.1 _always_ has annotation processing enabled.  There is certainly no checkbox for it in the compiler settings.

```
Lombok Requires Annotation Processing
Annotation processing seems to be disabled for the project "AtixPos".
For  plugin to function correctly, please enable it under
"Settings > Build > Compiler > Annotation Processors"
Click on this notification to go to Settings now. (show balloon)
```

**Comment by dsjstc** [comment]
More meaningful title.

**Comment by AlexejK** [comment]
So what you are saying is that Android Studio has different configuration pages compared to upstream IntelliJ?

I personally do not use Android studio so this did not even occur to me that they would change so much. As for "always enabled" - our check is done through APIs of IDEA to check if it's enabled or not - and if you are getting the warning it would seem that this is __not enabled__, at least not in the IDEA standard way.

**Comment by dsjstc** [comment]
The pages are indeed different.  Here's a [screenshot of the Android Studio compiler settings](http://imgur.com/LHWAlMf).  NB the complete lack of subpages.

I will post a bug against Studio too.  Are you able to easily tell me what the API is that you use to test enabled?

**Comment by AlexejK** [comment]
Interesting indeed. We are looking at `CompilerConfiguration.getAnnotationProcessingConfiguration()` or all modules in the project to check if it's enabled or not.

**Comment by dsjstc** [comment]
[Cross-reported to Android Studio.](https://code.google.com/p/android/issues/detail?id=218386&q=autumn%40thundersun.ca&colspec=ID%20Status%20Priority%20Owner%20Summary%20Stars%20Reporter%20Opened#makechanges)

**Comment by kamilZ** [comment]
I found it on:
File > Other Settings > Default Settings > Build, Execution, Deployment > Compiler > Annotation Processors > Check "Enable annotation processing"

**Comment by fernandospr** [comment]
@kamilZ I have enabled it, however the error message is still shown when opening Android Studio.

**Comment by wolkenschieber** [comment]
The error message still appears on Android Studio 3.0.1.

As mentioned by @kamilZ, the setting can accessed through *Other Settings*. But even with enabled option, the error persists.

Is there some additional name to be set?

**Comment by akramul-hasan-m4** [comment]
### Problem is alive with android studio 3.3.2 and lombok version: 0.24-2018.2


![lombok](https://user-images.githubusercontent.com/34574028/55528174-c4ad2500-56bd-11e9-9435-920fc33ec333.png)
)

**Comment by akramul-hasan-m4** [comment]
### Finally i found a solution , it is 100% works for me.
1. Go to File -> Other Settings -> Settings for new projects -> Build, Execution, Deployment -> Compiler ->Annotation Processors
2. Check, Enable Annotation Processing
3. Click Apply 
4. Restart Your Android studio.

### I Have Existing Project, What can I do?
1.  Go to your project folder then delete yourProject.iml , app.iml file and .idea folder
2.  Then re-import your project

*Hope your problem will solve*

**Comment by mplushnikov** [comment]
@akramul-hasan-m4 Thank you for your suggestion!

**Comment by mplushnikov** [comment]
Will be fixed in the next Build, 0.29. By PR from @wyhasany https://github.com/mplushnikov/lombok-intellij-plugin/pull/744
